compiler/rustc_const_eval/src/const_eval/machine.rs

```diff
@@ -3,6 +3,7 @@ use std::fmt;
 use std::hash::Hash;
 use std::ops::ControlFlow;
 
+use rustc_abi::{Align, ExternAbi as CallAbi, Size};
```

Comment:
FWIW feel free to drop the `as CallAbi`, we had it to distinguish from the type now known as `BackendRepr`.

We can also do that later, though, if you'd prefer.

Reply:
Oh, in general avoiding that diff was out of caution, so I'll go ahead and apply that.